pkg/api/helpers.go

```diff
@@ -350,6 +351,10 @@ func CreateServer(api *ScalewayAPI, c *ConfigCreateServer) (string, error) {
 		return "", errors.New("Invalid commercial type")
 	}
 
+	if !(c.BootType == "local" || c.BootType == "bootscript") {
```

Comment:
not a big issue, but `switch` is more appropriate here, or at last the same condition without the global `!`, i.e.: `if c.BootType != "local" && c.BootType != "bootscript"`